frontend/packages/policy-editor/src/components/PolicyEditor.tsx

```diff
@@ -217,8 +234,13 @@ export const PolicyEditor = ({
   return (
     <div>
       <div className={classes.alertWrapper}>
-        <Alert iconTitle='Du må ha minimum en regel for å publisere ressursen.' severity='info'>
-          Du må ha minimum en regel for å publisere ressursen.
+        <Alert
+          iconTitle={`Du må ha minimum en regel for å publisere ${
+            usageType === 'app' ? 'appen' : 'ressursen'
+          }.`}
+          severity='info'
+        >
+          Du må ha minimum en regel for å publisere {usageType === 'app' ? 'appen' : 'ressursen'}.
```

Comment:
Nothing in the PolicyEditor handles language yet, and I think it would be wiser to separate the two tasks as mentioned above. 
The task is already included in the issue: https://github.com/Altinn/altinn-studio/issues/10746